db/db.js

```diff
@@ -294,6 +294,52 @@ async function updateSessionStillCounter(stillcounter, sessionid, locationid) {
   }
 }
 
+async function isOverdoseSuspectedInnosent(session, location) {
```

Comment:
I suggest that we add `clientParam` to all our new functions in `db.js` so that we'll be able to use them in transactions without thinking about it.